web/src/main/java/org/cbioportal/web/StudyViewController.java

```diff
@@ -87,8 +85,9 @@ private void init() {
     @Autowired
     private ClinicalDataBinUtil clinicalDataBinUtil;
     @Autowired
+    private AlterationCountService alterationCountService;
+    @Autowired
```

Comment:
Is this service being used here? I don't see any reference, maybe we can delete it.